modules/test/playwright/tests/export-import-web/site.import.spec.ts

```diff
@@ -358,9 +358,10 @@ test('can see corresponding elements at site level', async ({
 		exportImportPage.page.getByLabel('Delete Application Data')
 	).toBeVisible();
 
-	await expandSection(
-		exportImportPage.page.getByRole('button', {name: 'Update Data'})
-	);
+	await reloadImportUntilFieldsetOpened({
```

Comment:
As we agreed, maybe better to create two different methods, one to reload the import page and the other one to reload until field set is opened